src/binder/bind/copy/bind_copy_from.cpp

```diff
@@ -132,6 +132,19 @@ std::unique_ptr<BoundStatement> Binder::bindCopyRelFrom(const parser::Statement&
         expectedColumnTypes, clientContext);
     auto boundSource = bindScanSource(copyStatement.getSource(),
         copyStatement.getParsingOptionsRef(), expectedColumnNames, expectedColumnTypes);
+    expression_vector warningDataExprs;
+    bool ignoreErrors = CopyConstants::DEFAULT_IGNORE_ERRORS;
+    if (boundSource->type == ScanSourceType::FILE) {
+        auto& source = boundSource->constCast<BoundTableScanSource>();
+        auto bindData = source.info.bindData->constPtrCast<ScanBindData>();
+        for (column_id_t i = bindData->numWarningDataColumns; i >= 1; --i) {
+            KU_ASSERT(i < source.info.columns.size());
+            warningDataExprs.push_back(source.info.columns[source.info.columns.size() - i]);
+        }
+
+        ignoreErrors =
+            bindData->config.getOption(CopyConstants::IGNORE_ERRORS_OPTION_NAME, ignoreErrors);
+    }
```

Comment:
See if this can be consolidated since code like this appears in multiple places

Reply:
This one is harder to hide because `warningDataExprs` needs to be passed to `IndexLookupInfo`

Reply:
removed by making warning data a common concept in `BoundScanSource`